pkg/runner/runner.go

```diff
@@ -135,31 +143,65 @@ func selectPlan(spec *bundle.Spec) bundle.Plan {
 	return bundle.Plan{}
 }
 
-func selectParameters(plan bundle.Plan) bundle.Parameters {
+func selectParameters(plan bundle.Plan) (bundle.Parameters, error) {
+	schemaPlan, err := bundle.ConvertPlansToSchema([]bundle.Plan{plan})
+	if err != nil {
+		log.Errorf("Error converting bundle plans to JSON Schema: %v", err)
+		return nil, err
+	}
+	planSchema := schemaPlan[0].Schemas
+	schemaParams := planSchema.ServiceInstance.Create["parameters"]
 	params := bundle.Parameters{}
 	for _, param := range plan.Parameters {
-		var paramDefault string
-		var paramInput string
+		var paramDefault interface{}
 		if param.Default != nil {
-			paramDefault = param.Default.(string)
+			paramDefault = param.Default
 		}
 		check := 1
```

Comment:
This line wasn't changed in your PR, but I'm wondering why this is an integer instead of a bool if it's just indicating whether input was checked.

Reply:
In case it's unclear, I'm referring to the `check` integer. Set to `1` signifying unchecked, or `2` signifying checked. Definitely a nit but seems like it should be a well-named bool that wouldn't require guessing at its purpose.

Reply:
+1 good idea